build/libraries/git-skip.js

```diff
@@ -0,0 +1,169 @@
+const fsUtil = require('./fs')
+const fs = require('fs')
+const path = require('path')
+const child_process = require('child_process')
+
+const DIR_GITROOT = path.join(fsUtil.WEBROOT, 'concrete')
+
+if (process.argv.indexOf('-h') > 1 || process.argv.indexOf('--help') > 1 || process.argv.indexOf('/?') > 1) {
+    exitWithSyntax(0)
+}
+
+setSkipWorktree(getSkippablePaths(), getSkipOption())
+
+function exitWithSyntax(returnCode)
+{
+    console.log(`Syntax: ${fsUtil.escapeShellArg(process.argv[0])} ${fsUtil.escapeShellArg(process.argv[1])} <0|no|off|false|1|yes|on|true>`)
+    process.exit(returnCode)
+}
+
+function getSkipOption()
+{
+    if (process.argv.length !== 3) {
+        exitWithSyntax(1)
+    }
+    switch (process.argv[2].toLowerCase()) {
+        case '0':
+        case 'no':
+        case 'off':
+        case 'false':
+            return false
+        case '1':
+        case 'yes':
+        case 'on':
+        case 'true':
+            return true
+    }
+    exitWithSyntax(1)
+}
+
+function getSkippablePaths()
+{
+    return [
+        'blocks/gallery/auto.js',
+        'css/ckeditor/concrete.css',
+        'css/cms.css',
+        'css/features/basics/frontend.css',
+        'css/features/boards/frontend.css',
+        'css/features/calendar/frontend.css',
+        'css/features/conversations/frontend.css',
+        'css/features/documents/frontend.css',
+        'css/features/express/frontend.css',
+        'css/features/faq/frontend.css',
+        'css/features/imagery/frontend.css',
+        'css/features/maps/frontend.css',
+        'css/features/multilingual/frontend.css',
+        'css/features/navigation/frontend.css',
+        'css/features/polls/frontend.css',
+        'css/features/search/frontend.css',
+        'css/features/social/frontend.css',
+        'css/features/taxonomy/frontend.css',
+        'css/features/testimonials/frontend.css',
+        'css/features/video/frontend.css',
+        'css/fontawesome/all.css',
+        'css/fullcalendar.css',
+        'images/icons/bedrock/sprites.svg',
+        'js/ckeditor/concrete.js',
+        'js/cms.js',
+        'js/features/boards/frontend.js',
+        'js/features/calendar/frontend.js',
+        'js/features/conversations/frontend.js',
+        'js/features/documents/frontend.js',
+        'js/features/express/frontend.js',
+        'js/features/imagery/frontend.js',
+        'js/features/maps/frontend.js',
+        'js/features/multilingual/frontend.js',
+        'js/features/navigation/frontend.js',
+        'js/fullcalendar.js',
+        'js/jquery.js',
+        'js/vue.js',
+        'themes/concrete/main.css',
+        'themes/concrete/main.js',
+        'themes/dashboard/main.css',
+        'themes/dashboard/main.js',
+        'themes/elemental//main.js',
```

Comment:
Double forward slash here. Looks like `git update-index` normalizes this but wanted to point it out in case you thought it's worth fixing

Reply:
Fixed by https://github.com/concrete5/concrete5/pull/9089/commits/0e16464591c53506bef53fa603fe3c5ab63f30b9